force-app/main/default/flows/Chatter_Digest_Email_frequency.flow-meta.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do you check your work once you done with it. Is this flow is achieving for which its intended ?